app/containers/HomeSettings/index.js

```diff
@@ -173,6 +190,63 @@ class SettingsPage extends Component<Props> {
     url: null
   };
 
+  removePath = async (event, label) => {
+    event.stopPropagation();
+    const { removePath, selectedPath, pathsList } = this.props;
+    const labelToRemove = pathsList.find(path => path.get('label') === label);
+    if (labelToRemove) {
+      if (label === selectedPath) {
+        await removePath(label);
+        if (pathsList.size > 2) {
+          const parser = JSON.parse(localStorage.settings);
+          const listLength = parser.pathsList.length;
+          const labelToAdd = parser.pathsList[listLength - 1].label;
+          this.handlePathChange(labelToAdd);
+        } else {
+          await this.handlePathChange('Default');
+        }
+      } else {
+        removePath(label);
+      }
+    }
+  };
+
+  removeNode = async (event, name) => {
```

Comment:
There are some issues on this function.
if (JSON.parse(localStorage.settings).nodesList.length > 4) {
      await this.handleConseilChange('Cryptonomic-Conseil');
} else {
       this.handleConseilChange('Cryptonomic-Conseil');
 }

Reply:
ah yes, good catch I've cleaned that function up now.